service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -82,6 +88,23 @@ public void sendNotificationForC2Upload(final C2UploadedEvent caseEvent) {
         }
     }
 
+    @EventListener
+    public void sendNotificationForC21Order(final C21OrderEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = (String) caseDetails.getData().get(CASE_LOCAL_AUTHORITY_PROPERTY_NAME);
+
+        Map<String, Object> parameters = c21OrderEmailContentProvider.buildC21OrderNotification(
+            caseDetails, localAuthorityCode);
+        String reference = Long.toString(caseDetails.getId());
+
+        String localAuthorityEmail = hmctsCourtLookupConfiguration.getCourt(localAuthorityCode).getEmail();
+        String cafcassEmail = cafcassLookupConfiguration.getCafcass(localAuthorityCode).getEmail();
+
+        Stream.of(localAuthorityEmail, cafcassEmail)
+            .filter(StringUtils::isNotBlank)
```

Comment:
will either of these emails ever be null? I don't think so...

Reply:
I agree it should not. But no harm checking was my thinking.

Reply:
I think the local authority email MAY be null. We have got some follow up stories for SDO, should the same behaviour be applied here? I would say so, please check with BAs

Reply:
Hmmmm... was not aware of this @akomisarek, will speak to a BA to clarify how we proceed.